org-chart/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,69 @@
+import { useContext } from "react";
+import './Card.css';
+import { AvatarContext } from "../../contexts/AvatarContext";
+import { IoPeople } from 'react-icons/io5';
```

Comment:
```suggestion
import { useContext } from "react";
import { IoPeople } from 'react-icons/io5';

import './Card.css';
import { AvatarContext } from "../../contexts/AvatarContext";
```

Usually separate package imports from relative imports. Packages come first, before relative ones.